package.json

```diff
@@ -14,12 +14,14 @@
     "lint:fix": "./node_modules/.bin/eslint . --fix"
   },
   "dependencies": {
-    "bunyan": "^1.8.9",
-    "shr-models": "^5.2.1"
+    "bunyan": "^1.8.9"
   },
   "devDependencies": {
     "chai": "^3.5.0",
     "eslint": "^3.6.1",
     "mocha": "^3.2.0"
+  },
+  "peerDependencies": {
+    "shr-models": "^5.2.0"
```

Comment:
I know that peerDeps are supposed to be [loose](https://nodejs.org/en/blog/npm/peer-dependencies/#using-peer-dependencies), however this standardhealth/shr-models#15 bugfix directly affects at least one of the tests. Could we do `^5.2.1` instead? Or if we really want to drink the semver koolaid we could do `>= 5.2.1 < 6`.

Reply:
I didn't realize that v5.2.1 affected one of the tests.  Yeah, in that case we can do `^5.2.1` (which, according to my understanding, is exactly equivalent to `>= 5.2.1 < 6`).